[TM] Update vector classification lesson (part 2) (#5768)
qgis/QGIS-Documentation: docs/training_manual/vector_classification/classification.rst

#. Click the |signPlus| :sup:`Add rule` button
#. The :guilabel:`Edit rule` dialog then appears
#. Enter ``Swellendam city`` as :guilabel:`Label`
#. Click the |expression| button next to the :guilabel:`Filter` text
   area to open the :guilabel:`Expression String Builder`
#. Enter the criterion ``"name" = 'Swellendam'`` and validate

   .. figure:: img/query_builder_example.png
      :align: center

#. Back to the :guilabel:`Edit rule` dalog, assign it a darker
   grey-blue color in order to indicate the town's
   importance in the region and remove the border

   .. figure:: img/rule_style_result.png
      :align: center

#. Press :guilabel:`OK`
#. Repeat the steps above to add the following rules:

   #. **Other residential** label with the criterion
      ``"landuse" = 'residential' AND "name" <> 'Swellendam'`` (or
      ``"landuse" = 'residential' AND "name" != 'Swellendam'``). 
      Choose a pale blue-grey :guilabel:`Fill color`
   #. **Big non residential areas** label with the criterion
      ``"landuse" <> 'residential' AND "AREA" >= 605000``.
      Choose a mid-green color.

      .. figure:: img/criterion_refined_midway.png
         :align: center

>       These filters are exclusive, in that they collectively exclude
>       some areas on the map (i.e. those which are smaller than 605000
>       (square meters), are not residential, and are not
>       '|majorUrbanName|').

Reviewer: ```suggestion
      These filters are exclusive, in that they exclude areas on the
      map (non-residential areas which are smaller than 605000
      (square meters) are not included in any of the rules).
```
